MvvmCross/Platform/Tvos/Views/MvxTabBarViewController.cs

```diff
@@ -77,6 +77,12 @@ protected virtual void SetTitleAndTabBarItem(UIViewController viewController, Mv
 
         public virtual bool ShowChildView(UIViewController viewController)
         {
+            if (SelectedIndex > 5) // when more menu item is currently visible, selected index has value higher than 5
```

Comment:
Is there any variable available instead of hardcoding 5?

Reply:
There is no any constant there. We should remove the code for tvOS -> looks like it is not supported there.
We need to check how many tabs can be displayed in horizontal iOS mode / iPads as it might break child presenter on those devices.